src/components/LayoutBase/LayoutBase.tsx

```diff
@@ -16,6 +16,33 @@ type LayoutBaseProps = {
 export const LayoutBase = ({ children }: LayoutBaseProps) => {
   useNavbarStyles();
 
+  useEffect(() => {
+    const videoAskScript = document.createElement('script');
+    videoAskScript.innerHTML = `
+      window.VIDEOASK_EMBED_CONFIG = {
+        "kind": "widget",
+        "url": "https://www.videoask.com/fno6cfau6",
+        "options": {
+          "widgetType": "VideoThumbnailExtraLarge",
+          "text": "Hey!&#x1FAE1;",
+          "backgroundColor": "#00ABFF",
+          "position": "bottom-right",
+          "dismissible": true
+        }
+      };
+    `;
+    document.body.appendChild(videoAskScript);
+
+    const embedScript = document.createElement('script');
+    embedScript.src = 'https://www.videoask.com/embed/embed.js';
+    document.body.appendChild(embedScript);
+
+    return () => {
+      document.body.removeChild(videoAskScript);
+      document.body.removeChild(embedScript);
+    };
+  }, []);
+
```

Comment:
Isn't the same as add the:

```
import Head from '@docusaurus/Head';

      <Head>
        <script src="https://www.videoask.com/embed/embed.js" />
        <script>
          {`
      window.VIDEOASK_EMBED_CONFIG = {
        "kind": "widget",
        "url": "https://www.videoask.com/fno6cfau6",
        "options": {
          "widgetType": "VideoThumbnailExtraLarge",
          "text": "Hey!&#x1FAE1;",
          "backgroundColor": "#00ABFF",
          "position": "bottom-right",
          "dismissible": true
        }
      };
    `}
        </script>
      </Head>
```

In the component return? If yes, let's create a component analogical to `SEO.tsx` and let's just add it here.